docs/dashboards/data-quality-dashboard.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[data-quality-dash]]
+= Data Quality dashboard
+
+The Data Quality dashboard shows you whether your data is correctly mapped to the https://www.elastic.co/guide/en/ecs/current/ecs-reference.html[Elastic Common Schema] (ECS). Successful {ref}/mapping.html[mapping] enables you to search, visualize, and interact with your data throughout {es-sec} and {kib}.
+
+image::images/data-qual-dash.png[The Data Quality dashboard]
+
+You can use the Data Quality dashboard to:
+
+* Check one or multiple indices for unsuccessful mappings, to help you identify problems (the indices used by {es-sec} appear by default).
+* View detailed information about the fields in checked indices.
+* Track unsuccessful mappings by creating a case or Markdown report based on data quality results.
+
+NOTE: The Data Quality dashboard does not show data from cold or frozen {ref}/data-tiers.html[data tiers]. It also does not display data from remote clusters using cross-cluster search — to view data from another cluster, log in to that cluster's {kib} instance.
+
+.Requirements
+[sidebar]
+--
+To use the Data Quality dashboard, you need at least the following {ref}/security-privileges.html#privileges-list-indices[permissions] for each index you want to check:
+
+. `monitor` or `manage` (required for the {ref}/indices-stats.html[Index stats API])
+. `view_index_metadata` or `manage_ilm` (required for the {ref}/ilm-explain-lifecycle.html[Explain lifecycle API])
+. `view_index_metadata` or `manage` (required for the {ref}/indices-get-mapping.html[Get mapping API])
+. `read` (required for the {ref}/search-search.html[Search API])
```

Comment:
Only use numbered steps when things need to be done in a specific order; bullets are more appropriate for a list like this. I'll leave the API statements untouched for now, but see my question above about whether they're necessary.
```suggestion
* `monitor` or `manage` (required for the {ref}/indices-stats.html[Index stats API])
* `view_index_metadata` or `manage_ilm` (required for the {ref}/ilm-explain-lifecycle.html[Explain lifecycle API])
* `view_index_metadata` or `manage` (required for the {ref}/indices-get-mapping.html[Get mapping API])
* `read` (required for the {ref}/search-search.html[Search API])
```